src/VisualStudio/Core/Def/Implementation/AnalyzerDependencyChecker.cs

```diff
@@ -46,7 +46,7 @@ public AnalyzerDependencyResults Run(CancellationToken cancellationToken = defau
                 }
             }
 
-            _assemblyWhiteLists.Add(new AssemblyIdentityWhiteList(analyzerInfos.Select(info => info.Identity)));
+            _ignorableAssemblyLists.Add(new IgnorableAssemblyIdentityList(analyzerInfos.Select(info => info.Identity)));
```

Comment:
please change to `_skippableAssemblyList.Add(new SkippableAssemblyIdentityList(analyzerInfos.Select(info => info.Identity)));`